Casks/activitywatch.rb

```diff
@@ -3,12 +3,21 @@
   sha256 "91deabd5844968467b5b6b3423e12654d4e339ba309f333c25d991a2e689fd96"
 
   url "https://github.com/ActivityWatch/activitywatch/releases/download/v#{version}/activitywatch-v#{version}-macos-x86_64.dmg",
-      verified: "github.com/ActivityWatch/activitywatch/"
+      verified: "github.com/ActivityWatch/activitywatch/releases/download/"
   name "ActivityWatch"
   desc "Time tracker"
   homepage "https://activitywatch.net/"
 
+  livecheck do
+    url "https://activitywatch.net/downloads/"
+    regex(%r{href=.*?/activitywatch-v([0-9]+(\.[0-9]+)+)-macos-x86_64\.dmg}i)
```

Comment:
@p-linnane In line with the [guidelines](https://docs.brew.sh/Brew-Livecheck) - here's some formatting adjustments to bring the regex in line with the standards.
```suggestion
    regex(/href=.*?activitywatch[._-]v?(\d+(?:\.\d+)+)-macos-x86_64\.dmg/i)
``

Reply:
@bevanjkay Thank you. I didn't realize there were specific regex guidelines. Will adjust and utilize moving forward.